webapp/src/components/views/BackendProbe.tsx

```diff
@@ -44,14 +44,17 @@ export const BackendProbe: FC<IData> = ({ uri, onBackendFound }) => {
                     return;
                 }
 
-                const json: unknown = await result.json();
-
-                if (json === null) {
-                    dispatch(setMaintenance(false));
-                    onBackendFound();
-                }
-
-                model.current = json as IMaintenance | null;
+                result
+                    .json()
+                    .then((data) => {
+                        // Has payload is maintenance
+                        setModel(data as IMaintenance);
+                    })
+                    .catch(() => {
```

Comment:
This catch block is meant catch exceptions. If the API errors, it doesn't necessarily mean that the backend is not in maintenance mode and it would prematurely bail out of this health check. 

From the code in `MaintenanceController` above, it seems like the only condition where the backend is not in maintenance mode is if the API returns a 200 OK with no body, so shouldn't this bebe in the then block above? Under a conditional that checks for any empty body
```
if (!data)
{
 // No maintenance payload
                        dispatch(setMaintenance(false));
                        onBackendFound();
}
```

Reply:
The exception is the JSON parsing of undefined body.  Since the body is a stream, its a bit indirect to evaluate up front...hence, the response to the error.  I believe the API has returned response already at this point.

Reply:
Ah I see now, can you add a comment to clarify the JSON exception we're catching here?